src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState.cs

```diff
@@ -75,7 +75,7 @@ internal sealed partial class SolutionCompilationState
         _frozenSourceGeneratedDocumentState = frozenSourceGeneratedDocument;
 
         // when solution state is changed, we recalculate its checksum
-        _lazyChecksums = AsyncLazy.Create(c => ComputeChecksumsAsync(projectsToInclude: null, c));
+        _lazyChecksums = AsyncLazy.Create(c => ComputeChecksumsAsync(projectId: null, c));
```

Comment:
we don't need projectsToInclude as the data we incorporate into the solution-compilation-state is not affected by the set of projects included, only the single project we ask to sync.

when we get to the *solution state* however, this bubbles into N projectsToInclude.  NAmely the transitive set of proejcts taht the sync'ed project depends on .  

so you'll stills see ComputeChecksumsAsync(projectsToInclude) in SolutionState, just not in SolutionCompilationState.